### PR TITLE
Fix for Teensy RESET key.

### PR DESCRIPTION
**Pull request by aspickard**
Let me know if there is a better way to declare the TEENSY config option.

### CODE REVIEW

File: quantum/keymap_common.c [hunk 1]
```diff
@@ -74,7 +74,21 @@ action_t action_for_key(uint8_t layer, keypos_t key)
     } else if (keycode == RESET) { // RESET is 0x5000, which is why this is here
         clear_keyboard();
         _delay_ms(250);
+// bootloader_jump doesn't work for the Teensy
+#ifdef TEENSY
```
Comment (Rovanion) [inline quantum/keymap_common.c]: I found this code to be a straight copy of the ´#ifdef __AVR_ATmega32U4__´ in ´bootloader_jump()´ in [tmk_core/common/avr/bootloader.c](https://github.com/jackhumbert/qmk_firmware/blob/master/tmk_core/common/avr/bootloader.c). As of such these changes can be reduced to what's defined in PR #162.

Comment (jackhumbert) [inline quantum/keymap_common.c]: I believe the line you're referring to is commented out via the `#if 0` [here](https://github.com/jackhumbert/qmk_firmware/blob/master/tmk_core/common/avr/bootloader.c#L117).

**Comment by jackhumbert** [comment]
This looks good, but seems to be conflicting with master - would you mind pull the latest updates?

**Comment by aspickard** [comment]
@jackhumbert After some testing I have found that #162 fixes the problem much more succinctly - not sure why.

**Comment by jackhumbert** [comment]
Interesting! Let's move this over to #162 then.

**Pull request closed without merging**